pkg/cmd/bazci/githubpost/issues/condense.go

```diff
@@ -55,6 +55,13 @@ type FatalNodeRoachtest struct {
 	FatalLogs string
 }
 
+// NodeToIpMappingRoachtest contains the node to ip mapping from a roachtest
+// cluster
+type NodeToIpMappingRoachtest struct {
```

Comment:
Nit: the naming convention below uses `roachtest` as prefix vs. the above suffix.